config/_ytt_lib/github.com/cloudfoundry/metric-proxy/config/manifests/0-metrics-server-components-v0.3.6.yml

```diff
@@ -0,0 +1,157 @@
+#@ load("@ytt:data", "data")
+
+#@ if data.values.metric_proxy.enable_metrics_server:
```

Comment:
What was the motivation behind packaging all of these RBAC resources directly in metric-proxy instead of the config-optional file?

I worry whether the config-optional file functions without any of these permissions, but maybe I could see this organized as a separate file that works with metrics-server on any platform? So this file remains constant while the other config-optional file may change image or something based on the platform you have to install metrics-server it on?

Reply:
Didn't think to package it in config-optional since that seemed to be a place for configs that mostly only apply to Kind environments. 

As was tested above here: https://github.com/cloudfoundry/cf-for-k8s/pull/213#discussion_r434881569, this shouldn't be applied if the metrics-server isn't added directly in the config dir. It'll fail the `ytt` compilation step.